app/views/notes/show.html.erb

```diff
@@ -61,9 +62,10 @@
   <%= render :partial => "notes/gallery" %>
 
   <div id="content" class="pl-content note">
-  <%= raw auto_link(insert_extras(@node.latest.render_body), :sanitize => false) %>
+  <%= raw auto_link(insert_extras(@node.latest.render_body || @body.render_body), :sanitize => false) %>
   </div>
 
+  <%# TODO8: Is this depreciated code? Ask @jywarren %>
```

Comment:
@jywarren

Reply:
Cool, I think we can remove this comment for now 👍